PowerPointLabs/PowerPointLabs/ActionFramework/NarrationsLab/RecordNarrations/RecordNarrationsActionHandler.cs

```diff
@@ -41,5 +37,26 @@ protected override void ExecuteAction(string ribbonId)
                 recorder.RecorderPaneReload();
             }
         }
+
+        private void TaskPaneVisibleValueChangedEventHandler(object sender, EventArgs e)
+        {
+            CustomTaskPane recorderPane = Globals.ThisAddIn.GetActivePane(typeof(RecorderTaskPane));
+
+            if (recorderPane == null)
+            {
+                return;
+            }
+
+            RecorderTaskPane recorder = recorderPane.Control as RecorderTaskPane;
+
+            // trigger close form event when closing hide the pane
+            if (!recorder?.Visible ?? false)
+            {
+                recorder.RecorderPaneClosing();
+                // remove recorder pane and force it to reload when next time open
+                // TODO: Callback to remove task pane from thisaddin, register event.
```

Comment:
Can summarize the list of TODOs in the PR?

Reply:
I will be removing this TODO in favor of the old one in this file.